assets/js/blocks/stock-filter/block.json

```diff
@@ -9,10 +9,7 @@
 		"html": false,
 		"multiple": false,
 		"color": {
-		  "link": true,
-		  "__experimentalDefaultControls": {
-			"text": true
-		  }
+		  "link": true
```

Comment:
I wonder if we should stick with `"color": true` here, the same way we do for the `rating-filter` block? We don't really have any links to customize on this block (unless I'm missing some use-case here). What do you think?

Reply:
Good catch! I thought it was used for the _Reset_ button, but it turns out that it's not applied there because its a `<button>` and not a link.

I removed it in 704408fc0836c9bdc20f28a713b7f3d18de1e700.